android/src/main/java/com/thebluealliance/androidclient/datafeed/status/TBAStatusController.java

```diff
@@ -110,6 +117,25 @@ public int getLatestAppVersion() {
         return status.getLatestAppersion();
     }
 
+    public void clearOkCacheIfNeeded(@Nullable APIStatus status, boolean forceClear) {
+        long lastCacheClear = mPrefs.getLong(STATUS_CACHE_CLEAR_KEY, Long.MIN_VALUE);
+        if (status != null
+                && mOkHttpCache != null
+                && (forceClear || lastCacheClear < status.getLastOkHttpCacheClear())) {
+            Schedulers.io().createWorker().schedule(() -> {
+                Log.i(Constants.LOG_TAG, "Clearing OkHttp cache");
+                try {
+                    mOkHttpCache.evictAll();
+                    mPrefs.edit()
+                            .putLong(STATUS_CACHE_CLEAR_KEY, System.currentTimeMillis() / 1000l)
```

Comment:
I read the number as "10001" at first glance. Probably best to use a capital L to avoid the 1/l ambiguity in monospace font.